include/database/DBManager.php

```diff
@@ -1876,7 +1879,7 @@ public function prepareQuery($sql)
      * @param  array $data The array of data to replace the tokens with.
      * @return resource result set or false on error
      */
-    public function executePreparedQuery($stmt, $data = array())
+    public function createPreparedQuery($stmt, $data = array())
```

Comment:
I don't think we'd be able to rename for backwards-compatibility reasons since this is a public function. We'd need to either keep it as ```executePreparedQuery``` or make a new ```createPreparedQuery``` function and deprecate this one.